articles/quickstart/native/android/03-session-handling.md

```diff
@@ -74,133 +108,67 @@ private final AuthCallback callback = new AuthCallback() {
 
     @Override
     public void onSuccess(@NonNull Credentials credentials) {
-        saveCredentials(credentials);
+        credentialsManager.saveCredentials(credentials);
         //...
     }
 };
 ```
 
 ::: note
-User credentials are stored in [Private mode](https://developer.android.com/reference/android/content/Context.html#MODE_PRIVATE) in the seed project in the `SharedPreferences` file. You can achieve this with the `CredentialsManager` class. For details, check the implementation in the project code. There are better and more secure ways to store tokens, but we will not cover them in this tutorial.
+The Storage implementation given to the Credentials Manager in the seed project uses a SharedPreferences file to store the user credentials in [Private mode](https://developer.android.com/reference/android/content/Context.html#MODE_PRIVATE). Change this behavior by implementing a custom Storage. 
```

Comment:
~Change this behavior by implementing a custom Storage.~
You can modify this behavior by implementing a custom Storage ??

What is Storage defined as, a class, an object etc ? Sounds like this should be added after.

Reply:
Is an interface that the manager requires to store/retrieve stuff. Users can implement their own logic or use the one we provide. I'll prepend some context and accept your suggestion for the last sentence.